pkgs/self-actuated/actuated-cli/registry.yaml

```diff
@@ -0,0 +1,23 @@
+packages:
+  - type: github_release
+    repo_owner: self-actuated
+    repo_name: actuated-cli
+    description: CLI for actuated
+    version_constraint: "false"
+    version_overrides:
+      - version_constraint: "true"
+        asset: actuated-cli
+        format: raw
+        overrides:
+          - goos: darwin
+            goarch: amd64
+            asset: actuated-cli-{{.OS}}
+          - goos: darwin
+            goarch: arm64
+            asset: actuated-cli-{{.OS}}-{{.Arch}}
+          - goos: linux
+            goarch: amd64
+            asset: actuated-cli
```

Comment:
```suggestion
```

Reply:
😇
Thank you for your suggestion. I fixed it.
https://github.com/aquaproj/aqua-registry/pull/17466/commits/cec94d074e75ade974899492666c464a004cdc67